api/src/main/java/org/hawkular/inventory/api/Inventory.java

```diff
@@ -26,4 +34,83 @@
 public interface Inventory extends AutoCloseable {
 
     Tenants.ReadWrite tenants();
+
+    /**
+     * Quickly navigate to the provided tenant.
+     *
+     * @param tenant the tenant to steer to.
+     * @return the access interface to the tenant
+     * @throws EntityNotFoundException if the tenant is not found
+     */
+    default Tenants.Single steerTo(Tenant tenant) throws EntityNotFoundException {
```

Comment:
Why default methods in the interface instead of old fashion abstract class? To me the default methods are good for adding new methods for class that already extends something without breaking the compatibility (e.g. `List.stream()`). 
Now, we have the luxury that no one is using the API so we can break whatever we want. I'd perhaps save the default methods for the future.